cmd/server/main.go

```diff
@@ -1,3 +1,24 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	"github.com/axelx/go-yandex-metrics/internal/config"
+	"github.com/axelx/go-yandex-metrics/internal/handlers"
+	"github.com/axelx/go-yandex-metrics/internal/storage"
+	"net/http"
+)
+
+func main() {
+	metricStorage := storage.New()
+
+	conf := config.NewConfigServer()
+
+	fmt.Println("Running server on", conf.FlagRunAddr)
+
+	hd := handlers.New(&metricStorage)
+	err := http.ListenAndServe(conf.FlagRunAddr, hd.Router())
```

Comment:
Не ошибка. Доп инфа, го позволяет делать так
```suggestion
	if err := http.ListenAndServe(conf.FlagRunAddr, hd.Router()); err!=nil{
	}
```